### DISCUSSION TITLE
DoT's on 2 or 3 targets question

### DISCUSSION THREAD

**Discussion by LEOMODE** [Hekili/hekili#1006] [Q&A]
Hi there, I know I have both single-target and multi-target modes in Hekili. I key-binded to switch manually in between both since I'm playing a ranged DPS and it doesn't detect it as fluid as melee DPS which is fine with me. However, I guess my question is, for instance on Warlock if you have just 2 or 3 targets, I know it's best to put DoT's on both or all of them. And I know there is a small arrow logo in Hekili primary window which means the buff is already there on that target. So does that mean Hekili is telling me to DoT both, or does that mean I just still follow whatever Hekili tells me on that particular target despite showing an arrow? For multi-target mode (i.e. 5 or more targets) I know that I just have to put like one DoT on my main target and just spam AOE skills for most of the time, but 2-3 targets is what I'm confused about (like Destruction Warlock, I have to use Havoc and I can only use it for two targets. So should I need to set multi-target as 2, or just default to 3?) I just want to know the mechanics of this add-on.

Thanks in advance!

**Comment by Hekili** [comment]
The little arrow appears if you should use the ability on a different target.  You can set your AOE targets to 2 or 3; Havoc will only be recommended with multiple targets or that single-target legendary.

**Reply by LEOMODE** [reply]
Got it, interesting that Havoc was only for 3 or more targets! One follow-up question is, if I do want to use it on 2 targets, how should I leverage Hekili? Because I can't have 3 options (single target, 2 targets, 3+ targets), should I just set the default to 2 targets so I can utilize Havoc? For DoT classes, if I have 2 targets I know it's optimal to apply DoT to both target, but if I am currently using a single-target mode, Hekili doesn't tell me to apply the DoT to the other second target so I'm curious. 

One suggestion about the little arrow also (or if you can clarify my suggestion that'll be helpful too): When I am casting already (for instance immolate), it shows the arrow already. Can it make it appear once the immolate is when it's actually applied to the target? Also want to tell you my dps did increase like 1000 because of your addon! :)

**Reply by LEOMODE** [reply]
Oh, and is Corruption useless in both Destruction and Demonology spec? I never saw it in Hekili....

**Reply by Hekili** [reply]
You can open the priority and see if/why it is in there.  Or sim yourself in Destro/Demo and see if your character would ever cast it.

**Reply by LEOMODE** [reply]
Thanks for your answer. I'll take a look. If you have time, it'll be very helpful if you can help me answering my perhaps most important question for your addon which was just above what you answered my last comment on:

If I do want to use it on 2 targets, how should I leverage Hekili? Because I can't have 3 options (single target, 2 targets, 3+ targets), should I just set the default to 2 targets so I can utilize Havoc? For DoT classes, if I have 2 targets I know it's optimal to apply DoT to both target, but if I am currently using a single-target mode, Hekili doesn't tell me to apply the DoT to the other second target so I'm curious._

I did look at QuickSim and it does look like Havoc is used for both 2 targets or multi-targets (3+). So I'm not sure if only destruction spec should set AOE as 2 targets to start with. Thanks.

**Reply by LEOMODE** [reply]
hi there, a gentle reminder for the above. Thank you so much for your help.

**Reply by Hekili** [reply]
If you want the AOE display to display recommendations for 2+ targets, yes, setting your AOE target threshold to 2 targets would be correct, yes.
